docs/src/main/asciidoc/stork-kubernetes.adoc

```diff
@@ -0,0 +1,479 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+= Getting Started with SmallRye Stork
+:extension-status: preview
+
+// Temporary until stork in in the BOM
+:stork-version: 1.1.0
+include::./attributes.adoc[]
+
+The essence of distributed systems resides in the interaction between services.
+In modern architecture, you often have multiple instances of your service to share the load or improve the resilience by redundancy.
+But how do you select the best instance of your service?
+That's where https://smallrye.io/smallrye-stork[SmallRye Stork] helps.
+Stork is going to choose the most appropriate instance.
+It offers:
+
+* Extensible service discovery mechanisms
+* Built-in support for Consul and Kubernetes
+* Customizable client load-balancing strategies
+
+include::./status-include.adoc[]
+
+== Prerequisites
+
+:prerequisites-docker:
+include::includes/devtools/prerequisites.adoc[]
+* Access to a Kubernetes cluster (Minikube is a viable option)
+
+== Architecture
+
+In this guide, we will work with a few components deployed in a Kubernetes cluster:
+
+* A simple blue service.
+* A simple red service.
+* The `color-service` is the Kubernetes service which is the entry point to the Blue and Red instances.
+* A REST Client calling the blue or red service (the selection is delegated to Stork). The REST Client is also deployed in a namespace within a Kubernetes cluster
```

Comment:
These two bullets are about pieces of the same service, right? Shouldn't we merge them? 
Also is the "is also deployed in a namespace..." information important, esp. if we say that they are all in the same namespace a few lines further?

Maybe to something like:
```
* A client service using a REST client to call the blue or the red service. Service discovery and selection are delegated to Stork. The client service is also deployed in the Kubernetes cluster.
```

Reply:
:+1: Yes good idea. Thanks